code/game/machinery/deployable.dm

```diff
@@ -104,6 +109,7 @@
 	icon = 'icons/obj/structures.dmi'
 	icon_state = "woodenbarricade"
 	material = WOOD
+	debris_type = /obj/item/stack/sheet/mineral/wood
```

Comment:
Someone had removed/forgotten to add the type of debris for wooden barricade. This fixes wood barricade not dropping any wood.